core/variant/variant.h

```diff
@@ -531,7 +531,8 @@ class Variant {
 	static bool is_builtin_method_const(Variant::Type p_type, const StringName &p_method);
 	static bool is_builtin_method_static(Variant::Type p_type, const StringName &p_method);
 	static bool is_builtin_method_vararg(Variant::Type p_type, const StringName &p_method);
-	static void get_builtin_method_list(Variant::Type p_type, List<StringName> *p_list);
+	static void get_builtin_method_list(Variant::Type p_type, List<StringName> &r_list);
+	static void get_builtin_static_method_list(Variant::Type p_type, List<StringName> &r_list);
```

Comment:
Is there any particular reason to change these from pointer to reference?

Reply:
I thought it was a good idea, since these functions are useless without passing a list parameter (and there is no place in code where it happened). Passing a reference is less error-prone than passing a pointer (which requires a null checking anyway).

Reply:
Removed this change.